coq/coq-compile-common.el

```diff
@@ -92,11 +94,38 @@ Must be used together with `coq-par-enable'."
 	ncpus
       nil)))
 
+(defvar coq-max-background-vio2vo-percentage-shadow 40
+  "Internal shadow value of `coq-max-background-vio2vo-percentage'.
+This variable does always contain the same value as
+`coq-max-background-vio2vo-percentage'. It is used only to break
+the dependency cycle between `coq-set-max-vio2vo-jobs' and
+`coq-max-background-vio2vo-percentage'.")
+
+(defvar coq-internal-max-vio2vo-jobs 1
+  "Internal number of vio2vo jobs.
+This is the internal value, use
+`coq-max-background-vio2vo-percentage' to configure.")
```

Comment:
Sure, but since when do we have this convention? IMO there are a lot internal variables in PG without --.